os/kernel/task/task_prctl.c

```diff
@@ -267,6 +272,81 @@ int prctl(int option, ...)
 		tcb->is_active = true;
 	}
 	break;
+#endif
```

Comment:
I don't understand why ioctl is needed???

Reply:
First of all, callback information needs to be stored in kernel space because app binary can't access another binary space each other in binary separation.
Also all preference operates with file operations like open, read, write and prctl.
It means user binary raises system interrupt many time for one operation because of system call.
So I think it'd better to be operated in kernel space with ioctl.

Reply:
please discuss with @sunghan-chang regarding what I told you.